pkg/api/schema/schema.proto

```diff
@@ -365,7 +365,13 @@ message TxHeader {
 }
 
 // TxMetadata contains metadata set to whole transaction
-message TxMetadata {}
+message TxMetadata {
+  // True if this metadata has a truncated transaction
+  bool hasTruncatedTxID = 1;
```

Comment:
probably `hasTruncatedTxID` may not be needed

Reply:
yes, makes sense, removed